src/sections/Meshery/meshery.style.js

```diff
@@ -66,7 +66,7 @@ const MesheryWrapper = styled.section`
     .callout{
         margin: auto;
         margin-top: 5rem;
-        text-align: right;
+        text-align: center;
```

Comment:
This change should happen only when the resolution is less than `768px`. Please write a media query to fix this issue.